cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/images/AddLaunchPermissionsToImageAction.java

```diff
@@ -136,7 +136,8 @@
                     .withUserGroupsString(userGroupsString)
                     .build();
 
-            return new QueryApiExecutor().execute(inputs, imageInputs);
+            Map<String, String> queryMapResult = new QueryApiExecutor().execute(inputs, imageInputs);
```

Comment:
@Tirla-Alin I have another comment (that is generic one): you put in every single action something like this: version = InputsUtil.getDefaultStringInput(version, "2016-04-01"); 
That's wrong because the 2016-04-01 is not a valid API version for every single section that is developed in this integration. For example in network area the latest API version available is: 2014-06-15. If you don't specify any API version then those operations will fail telling you that API is not available. So, you have to correct this issue in this PR because that modification is in master already.